lxd/device/unix_hotplug.go

```diff
@@ -0,0 +1,212 @@
+package device
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+   
+    deviceConfig "github.com/lxc/lxd/lxd/device/config"
+	"github.com/farjump/go-libudev"
+	"github.com/lxc/lxd/lxd/instance/instancetype"
+	"github.com/lxc/lxd/shared"
+)
```

Comment:
Our normal import order is:
 - built-in
 - external
 - internal

Each separate by a line break.
To match that, you should move the `go-libudev` to the external block, effectively making it:

```go
import (
	"fmt"
	"strconv"
	"strings"

	"github.com/farjump/go-libudev"

        deviceConfig "github.com/lxc/lxd/lxd/device/config"
	"github.com/lxc/lxd/lxd/instance/instancetype"
	"github.com/lxc/lxd/shared"
)
```

The `deviceConfig` line also looks misaligned. You should run `go fmt ./...` against your branch to fix such minor things.